packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1095,7 +1095,7 @@ export class ColorPicker
         slider: { width },
       },
     } = this;
-    const hue = (360 / width) * x;
+    const hue = ((HSV_LIMITS.h - 1) / width) * x;
```

Comment:
Nitpick: can we assign `1` to a self explanatory constant since its is used in other places or have a property which stores `HSV_LIMITS.h -1`